src/gui/footer/footer_line.cpp

```diff
@@ -41,6 +41,9 @@ bool FooterLine::Create(footer::Item item_id, size_t index) {
 
     if (item_id >= footer::Item::_count) {
         item_id = footer::Item::none;
+    } else if (item_id == footer::Item::none) {
+        unregister(index);
```

Comment:
This does not call positionWindows, which it probably should in some cases (at least according to the previous code).